content/cloud/custom-fermyon-subdomain.md

```diff
@@ -0,0 +1,69 @@
+title = "Apply Custom Fermyon Subdomain"
+template = "cloud_main"
+date = "2022-04-26T17:00:00Z"
+enable_shortcodes = false
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/custom-fermyon-subdomain.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Select Your Spin Application](#select-your-spin-application)
+- [Apply New Custom Fermyon Domain](#apply-new-custom-fermyon-domain)
+- [Validate Custom Fermyon Subdomain Name](#validate-custom-fermyon-subdomain-name)
+- [Next Steps](#next-steps)
+
+Every Spin application running on Fermyon Cloud receives a domain name that has the following format: `<your-App-Name-randomlyAssignedString>.fermyon.app`. For a more easily recognizable domain name, you may want to change your Spin application's domain name from `slats-the-cat-o7jecuug.fermyon.app` to `slatsthecat.fermyon.app`. 
+
+Custom Fermyon subdomain names allow you to rename the `<your-App-Name-randomyAssignedString>` subdomain. This custom Fermyon subdomain will be combined with the `.fermyon.app` apex domain to give your application a complete domain name. 
+
+## Prerequisites
+
+Log into [Fermyon Cloud](www.cloud.fermyon.com) and ensure you have an Spin application running on Fermyon Cloud. If you do not have a Spin application yet, follow our [quickstart guide](quickstart.md) to deploy one. 
```

Comment:
We want to have a complete URL otherwise it is going to be considered relative to the root of the website. Also minor grammatical nit.

```suggestion
Log into [Fermyon Cloud](https://www.cloud.fermyon.com) and ensure you have a Spin application running on Fermyon Cloud. If you do not have a Spin application yet, follow our [quickstart guide](quickstart.md) to deploy one. 
```